src/knapsack/BaseKnapsackModel.py

```diff
@@ -34,21 +32,15 @@ def getInputSpecs(cls):
     inputSpecs.addParam('name', param_type=InputTypes.StringType, required=True)
     inputSpecs.addParam('subType', param_type=InputTypes.StringType, required=True)
 
-    inputSpecs.addSub(InputData.parameterInputFactory('capacity', contentType=InputTypes.StringType))
     inputSpecs.addSub(InputData.parameterInputFactory('penaltyFactor', contentType=InputTypes.FloatType))
     inputSpecs.addSub(InputData.parameterInputFactory('outcome', contentType=InputTypes.StringType))
     inputSpecs.addSub(InputData.parameterInputFactory('choiceValue', contentType=InputTypes.StringType))
     inputSpecs.addSub(InputData.parameterInputFactory('variables', contentType=InputTypes.StringListType))
 
-    map = InputData.parameterInputFactory('map', contentType=InputTypes.StringType)
-    map.addParam('value', param_type=InputTypes.StringType, required=True)
-    map.addParam('cost', param_type=InputTypes.StringType, required=True)
-    inputSpecs.addSub(map)
-
-    alias = InputData.parameterInputFactory('alias', contentType=InputTypes.StringType)
-    alias.addParam('variable', param_type=InputTypes.StringType, required=True)
-    alias.addParam('type', param_type=InputTypes.StringType, required=True)
```

Comment:
I suggest to keep alias, this can be useful. The alias is handled directly by Model.py inside RAVEN without using InputData. If you want to keep this capability, we need to keep these three lines to let knapsack external model aware of it.

Reply:
I removed because I thought I merely did a copy and paste; restored